src/software/simulation/standalone_simulator_main.cpp

```diff
@@ -1,26 +1,29 @@
 #include "software/gui/standalone_simulator/threaded_standalone_simulator_gui.h"
 #include "software/logger/logger.h"
-#include "software/parameter/dynamic_parameters.h"
-#include "software/simulation/standalone_simulator.h"
+#include "software/simulation/threaded_simulator.h"
+#include "software/world/field.h"
 
-int main(int argc, char* argv[])
+int main(int argc, char *argv[])
```

Comment:
Could we remove the args here if we're not using them?

Reply:
TIL you don't need argc argv. Removed